2024-02-18 02:48:49.414 [info] (Client) Pylance async client (2024.2.2) started with python extension (2024.0.1)
2024-02-18 02:49:03.998 [info] [Info  - 2:48:59 AM] (736) Pylance language server 2024.2.2 (pyright version 1.1.348, commit cfb1de0c) starting
2024-02-18 02:49:04.119 [info] [Info  - 2:49:04 AM] (736) Server root directory: file:///root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist
2024-02-18 02:49:04.333 [info] [Info  - 2:49:04 AM] (736) Starting service instance "root"
2024-02-18 02:49:06.567 [info] [Info  - 2:49:06 AM] (736) Background analysis(1) root directory: file:///root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist
2024-02-18 02:49:06.597 [info] [Info  - 2:49:06 AM] (736) Background analysis(1) started
2024-02-18 02:49:06.673 [info] [Info  - 2:49:06 AM] (736) Setting pythonPath for service "root": "/usr/local/bin/python3"
2024-02-18 02:49:06.795 [info] [Info  - 2:49:06 AM] (736) Setting environmentName for service "root": "3.12.1 (グローバル)"
2024-02-18 02:49:07.510 [info] [Info  - 2:49:07 AM] (736) Assuming Python version 3.11
2024-02-18 02:49:32.776 [info] [Info  - 2:49:32 AM] (736) Found 43 source files
2024-02-18 02:49:35.432 [info] [Info  - 2:49:35 AM] (736) [BG(1)] Long operation: parsing: file:///root/manage.py (2327ms)
2024-02-18 02:49:35.761 [info] [Info  - 2:49:35 AM] (736) [BG(1)] Long operation: indexing: file:///root/manage.py (2655ms)
2024-02-18 04:25:12.736 [info] [Info  - 4:25:12 AM] (736) Indexer background runner(2) root directory: file:///root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist (index)
2024-02-18 04:25:12.765 [info] [Info  - 4:25:12 AM] (736) Indexing(2) started
2024-02-18 04:25:16.684 [info] [Info  - 4:25:16 AM] (736) Found 44 source files
2024-02-18 04:25:17.455 [info] [Info  - 4:25:17 AM] (736) [IDX(2)] Long operation: scan packages file:///root (4791ms)
2024-02-18 04:25:17.458 [info] [Info  - 4:25:17 AM] (736) scanned(2) 41 files over 1 exec env
2024-02-18 04:25:19.654 [info] [Info  - 4:25:19 AM] (736) [BG(1)] Long operation: analyzing: file:///root/general/views/login.py (2204ms)
2024-02-18 04:25:20.067 [info] [Info  - 4:25:20 AM] (736) indexed(2) 28 files over 1 exec env
2024-02-18 04:25:20.843 [info] [Info  - 4:25:20 AM] (736) Indexing finished(2).
2024-02-18 04:25:25.875 [info] [Info  - 4:25:25 AM] (736) Found 44 source files
2024-02-18 05:42:19.500 [info] [Info  - 5:42:19 AM] (736) Found 45 source files
2024-02-18 05:42:38.556 [info] [Info  - 5:42:38 AM] (736) Found 45 source files
2024-02-18 06:43:08.140 [info] [Info  - 6:43:08 AM] (736) Found 46 source files
2024-02-18 06:58:53.939 [info] [Info  - 6:58:53 AM] (736) Found 47 source files
2024-02-18 07:03:16.018 [info] [Info  - 7:03:15 AM] (736) Found 46 source files
2024-02-18 10:39:26.933 [info] [Info  - 10:39:26 AM] (736) Found 47 source files
2024-02-18 10:40:05.311 [info] [Info  - 10:40:05 AM] (736) Found 47 source files
2024-02-18 10:58:32.060 [info] [Info  - 10:58:32 AM] (736) Found 48 source files
2024-02-18 13:56:00.897 [info] [Error - 1:56:00 PM] (736) IntelliCode exception: Error: failed to call OrtRun(). ERROR_CODE: 2, ERROR_MESSAGE: Got invalid dimensions for input: input_batch:0 for the following indices
 index: 1 Got: 102 Expected: 100
 Please fix either the inputs or the model.
    at t.checkLastError (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:416241)
    at Object.t.run (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:411152)
    at t.run (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:400003)
    at t.OnnxruntimeWebAssemblySessionHandler.run (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:402850)
    at s.run (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:2527582)
    at _0x594aa0.<computed>.getRecommendations (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/server.bundle.js:1:423910)
    at _0x49cd1c.<computed>.addCompletionItems (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/server.bundle.js:1:436626)
    at /root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/server.bundle.js:1:913740
    at _0x1fa1f7.onCompletion (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/server.bundle.js:1:908352)
2024-02-18 13:56:02.618 [info] [Error - 1:56:02 PM] (736) IntelliCode exception: Error: failed to call OrtRun(). ERROR_CODE: 2, ERROR_MESSAGE: Got invalid dimensions for input: input_batch:0 for the following indices
 index: 1 Got: 102 Expected: 100
 Please fix either the inputs or the model.
    at t.checkLastError (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:416241)
    at Object.t.run (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:411152)
    at t.run (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:400003)
    at t.OnnxruntimeWebAssemblySessionHandler.run (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:402850)
    at s.run (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:2527582)
    at _0x594aa0.<computed>.getRecommendations (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/server.bundle.js:1:423910)
    at _0x49cd1c.<computed>.addCompletionItems (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/server.bundle.js:1:436626)
    at /root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/server.bundle.js:1:913740
    at _0x1fa1f7.onCompletion (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/server.bundle.js:1:908352)
2024-02-18 13:56:02.776 [info] [Error - 1:56:02 PM] (736) IntelliCode exception: Error: failed to call OrtRun(). ERROR_CODE: 2, ERROR_MESSAGE: Got invalid dimensions for input: input_batch:0 for the following indices
 index: 1 Got: 102 Expected: 100
 Please fix either the inputs or the model.
    at t.checkLastError (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:416241)
    at Object.t.run (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:411152)
    at t.run (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:400003)
    at t.OnnxruntimeWebAssemblySessionHandler.run (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:402850)
    at s.run (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:2527582)
    at _0x594aa0.<computed>.getRecommendations (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/server.bundle.js:1:423910)
    at _0x49cd1c.<computed>.addCompletionItems (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/server.bundle.js:1:436626)
    at /root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/server.bundle.js:1:913740
    at _0x1fa1f7.onCompletion (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/server.bundle.js:1:908352)
2024-02-18 14:06:11.411 [info] [Error - 2:06:11 PM] (736) IntelliCode exception: Error: failed to call OrtRun(). ERROR_CODE: 2, ERROR_MESSAGE: Got invalid dimensions for input: input_batch:0 for the following indices
 index: 1 Got: 102 Expected: 100
 Please fix either the inputs or the model.
    at t.checkLastError (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:416241)
    at Object.t.run (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:411152)
    at t.run (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:400003)
    at t.OnnxruntimeWebAssemblySessionHandler.run (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:402850)
    at s.run (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:2527582)
    at _0x594aa0.<computed>.getRecommendations (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/server.bundle.js:1:423910)
    at _0x49cd1c.<computed>.addCompletionItems (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/server.bundle.js:1:436626)
    at /root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/server.bundle.js:1:913740
    at _0x1fa1f7.onCompletion (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/server.bundle.js:1:908352)
2024-02-18 14:06:11.776 [info] [Error - 2:06:11 PM] (736) IntelliCode exception: Error: failed to call OrtRun(). ERROR_CODE: 2, ERROR_MESSAGE: Got invalid dimensions for input: input_batch:0 for the following indices
 index: 1 Got: 102 Expected: 100
 Please fix either the inputs or the model.
    at t.checkLastError (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:416241)
    at Object.t.run (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:411152)
    at t.run (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:400003)
    at t.OnnxruntimeWebAssemblySessionHandler.run (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:402850)
    at s.run (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:2527582)
    at _0x594aa0.<computed>.getRecommendations (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/server.bundle.js:1:423910)
    at _0x49cd1c.<computed>.addCompletionItems (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/server.bundle.js:1:436626)
    at /root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/server.bundle.js:1:913740
    at _0x1fa1f7.onCompletion (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/server.bundle.js:1:908352)
2024-02-18 14:06:12.050 [info] [Error - 2:06:12 PM] (736) IntelliCode exception: Error: failed to call OrtRun(). ERROR_CODE: 2, ERROR_MESSAGE: Got invalid dimensions for input: input_batch:0 for the following indices
 index: 1 Got: 102 Expected: 100
 Please fix either the inputs or the model.
    at t.checkLastError (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:416241)
    at Object.t.run (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:411152)
    at t.run (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:400003)
    at t.OnnxruntimeWebAssemblySessionHandler.run (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:402850)
    at s.run (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/vendor.bundle.js:2:2527582)
    at _0x594aa0.<computed>.getRecommendations (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/server.bundle.js:1:423910)
    at _0x49cd1c.<computed>.addCompletionItems (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/server.bundle.js:1:436626)
    at /root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/server.bundle.js:1:913740
    at _0x1fa1f7.onCompletion (/root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist/server.bundle.js:1:908352)
